src/Apps/Artist/Routes/Overview/state.ts

```diff
@@ -19,6 +19,23 @@ export class FilterState extends Container<State> {
     sort: "-partner_updated_at",
   }
 
+  constructor(props: any) {
```

Comment:
Should the input not just be `state: State` ?

Reply:
Ah of course.